pkg/container/hashtable/hash.go

```diff
@@ -99,6 +99,10 @@ func wyhash(p unsafe.Pointer, seed, s uint64) uint64 {
 	return mix(m5^s, mix(a^m2, b^seed))
 }
 
+func Wyhash64WithFixedSeed(x uint64) uint64 {
```

Comment:
all functions in this file are private except this newly added `Wyhash64WithFixedSeed`. Is this reasonable to set it as a public function?

Reply:
fixed